README.md

```diff
@@ -54,16 +54,16 @@ from confluent_kafka.avro.cached_schema_registry_client import CachedSchemaRegis
 from confluent_kafka.avro.serializer import util
 from confluent_kafka.avro.serializer.message_serializer import MessageSerializer
 
-value_avro_schema = util.parse_schema_from_file('ValueSchema.avsc')
-key_avro_schema = util.parse_schema_from_file('KeySchema.avsc')
+default_value_schema = util.parse_schema_from_file('ValueSchema.avsc')
+default_key_schema = util.parse_schema_from_file('KeySchema.avsc')
 value = {"name": "Value"}
 key = {"name": "Key"}
 
 producer = Producer({'bootstrap.servers': 'mybroker,mybroker2'})
 
 schema_registry_url = 'https://<host>:<port>'
 
-avroProducer = AvroProducer(producer, schema_registry_url, value_schema=value_avro_schema, key_schema=key_avro_schema)
+avroProducer = AvroProducer(producer, schema_registry_url, value_schema=default_value_schema, key_schema=default_key_schema)
```

Comment:
sorry, I meant name the pram `default_value_schema` and `default_key_schema`, not the values you pass in.

    avroProducer = AvroProducer(producer, schema_registry_url, default_value_schema=value_schema, default_key_schema=key_schema)